modules/appeals_api/spec/docs/v2/sc_spec.rb

```diff
@@ -7,9 +7,8 @@
 require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
 
 # rubocop:disable RSpec/VariableName, RSpec/ScatteredSetup, RSpec/RepeatedExample, Layout/LineLength, RSpec/RepeatedDescription
-describe 'Supplemental Claims', swagger_doc: 'modules/appeals_api/app/swagger/appeals_api/v2/swagger.json', type: :request do
+describe 'Supplemental Claims', swagger_doc: "modules/appeals_api/app/swagger/appeals_api/v2/swagger#{DocHelpers.doc_suffix}.json", type: :request, wip_doc_supplemental_claims: true do
```

Comment:
Here's where you can see the use of filter tagging. See `modules/appeals_api/spec/spec_helper.rb` for how that's hooked up.

Reply:
To clarify, this is a demonstration..._if_ supplemental claims were still in progress?

Reply:
Correct.